### PR TITLE
[TI] Update Sysconfig version from 1.16.2 to 1.18.1

### PR DESCRIPTION
**Pull request by abiradarti**
Updated documentation and docker image to reference a new sysconfig version

### CODE REVIEW

**Comment by abiradarti** [comment]
@andy31415 @bzbarsky-apple would you be able to assist with the new docker image creation here?

**Comment by bzbarsky-apple** [comment]
@abiradarti I don't have permissions to create new docker images, as far as I know.  That's @andy31415 and @woody-apple

File: integrations/docker/images/base/chip-build/version [hunk 1]
```diff
@@ -1 +1 @@
-43 : [NuttX] Fix build errors
```
Comment (andy31415) [inline integrations/docker/images/base/chip-build/version:1]: This has to be in a separate PR:

- one PR to change Dockerfiles + chip-build version
- a separate PR to make the changes in syscfg/readmes etc.

**Comment by andy31415** [comment]
> @andy31415 @bzbarsky-apple would you be able to assist with the new docker image creation here?

Please create a stand-alone separate PR to update the dockerfile changes only (2 docker files and one version file).

A separate PR then can update the rest and make use of the docker version 44 which I can build once the first PR is merged.

**Comment by abiradarti** [comment]
@andy31415 Thanks for the feedback, will separate this PR into two accordingly.
